go/server.go

```diff
@@ -250,6 +240,54 @@ func createPublicToken(c *gin.Context) {
 	})
 }
 
+func createLinkToken(c *gin.Context) {
+	linkToken, err := linkTokenCreate("")
+	if err != nil {
+		c.JSON(err.errorCode, gin.H{"error": err.error})
+	}
+	c.JSON(http.StatusOK, gin.H{"link_token": linkToken})
+}
+
+type httpError struct {
+	errorCode int
+	error     string
+}
+
+func (httpError *httpError) Error() string {
+	return httpError.error
+}
+
+// linkTokenCreate creates a link token using the specified parameters
+func linkTokenCreate(paymentID string) (string, *httpError) {
```

Comment:
nit: Perhaps we should accept a `*plaid.PaymentInitiation` struct here to set the precedence for other use-cases that may reuse this in the future.

Reply:
good idea